chicago/bills.py

```diff
@@ -197,7 +201,10 @@ def scrape(self, window=3) :
                     vote_event.add_source(legistar_api + '/histories')
 
                     for vote in votes :
-                        raw_option = vote['VoteValueName'].lower()
+                        vote_value = vote['VoteValueName']
+                        if vote_value is None:
```

Comment:
academic q: does a null vote represent anything? absence, e.g.?

Reply:
I don't know. this change does not reduce the amount of information we were recording, since we were excluding `None` vote_value before.